libopendmarc.so.2.0.3 uses
* __dn_expand
* __dn_skipname
* __res_nquery
which are defined in libresolv in glibc 2.33 and older.

See: https://bugs.gentoo.org/839951

--- a/libopendmarc/Makefile.am
+++ b/libopendmarc/Makefile.am
@@ -15,5 +15,6 @@
 			  opendmarc_spf_dns.c \
                           opendmarc_internal.h
 libopendmarc_la_LDFLAGS = -version-info $(LIBOPENDMARC_VERSION_INFO)
+libopendmarc_la_LIBADD = $(LIBRESOLV)
 libopendmarc_includedir = $(includedir)/opendmarc
 libopendmarc_include_HEADERS = dmarc.h
